api/src/controllers/search.js

```diff
@@ -325,7 +325,7 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
         defaultLog.info("err:", err);
         resolve([]);
       } else {
-        resolve(data);
+        resolve(data.toArray());
```

Comment:
The only thing I didnt look into fully was if resolve(new Promise()) is okay or not (.toArray() returns a promsie).  I know the entire function is being awaited, but not 100% sure if this is fine still.

I did some basic tests just in the web console and it seemed fine...

Reply:
This is ok